resources/views/pages/product/form.blade.php

```diff
@@ -0,0 +1,61 @@
+@extends('layouts.dashboard')
+@section('content')
+<div class="container mt-4">
+<h1>{{ $product->id ? "Edit" : "Create" }}</h1>
+@if ($product->id)
+    <form action="{{ route('product.update', ['product' => $product->id]) }}" method="post" enctype="multipart/form-data">
+    @method('PUT')
+@else
+    <form action="{{ route('product.store') }}" method="post" enctype="multipart/form-data">
+
+@endif
+    @csrf
+<div class="mb-3">
+    <label for="exampleFormControlInput1" class="form-label">Title</label>
+    <input type="text" class="form-control" name="title" id="exampleFormControlInput1" value="{{ $product->title }}">
+    @error('title') <div class="text-muted">{{ $message }}</div> @enderror
+  </div>
+  <div class="mb-3">
+    <label for="exampleFormControlInput1" class="form-label">Category Product</label>
```

Comment:
di bagian product/form.blade.php nomer 14 fornya sama dengan for 19,28 dsb seharusnya dibedakan